packages/inspector/public/css/app.css

```
@@ -451,3 +451,238 @@ main {
+
+#detail-pane {
+  flex: 0 0 320px;
+  overflow-y: auto;
+  background: var(--c-surface);
+  border-left: 1px solid var(--c-border);
+  display: flex;
+  flex-direction: column;
+  animation: slideIn 0.15s ease-out;
+}
+
+@keyframes slideIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)